Makefile

```diff
@@ -27,16 +27,21 @@ dist: $(VOUCH_DIST) $(VAULT_DIST)
 $(VOUCH_DIST): $(VOUCH_CODE)
 	cd $(SRCROOT) && \
 	rm -f dist/vouch* && \
-	python setup.py sdist && \
+	pip3 install setuptools && \
```

Comment:
Do you think its better to pin the setuptools version here?

Reply:
Probably but I was hoping to keep using the latest pip and setuptools. In fact, I may move this over to the common build image (assuming it has the right pip and setuptools installed) once it is available. The need to install setuptools is due to the odd base image that we have.

Reply:
I changed the image we were using and don't need to install setuptools explicitly anymore.